.github/workflows/pr-test.yml

```diff
@@ -67,6 +60,6 @@ jobs:
       - name: Archive integration test report
         uses: actions/upload-artifact@v1
         with:
-          name: integration-test
-          path: /tmp/integration-test-reports
+          name: test-reports
```

Comment:
Curious: Why are we uploading as artifacts when we are uploading to codecov?

Reply:
codecov gives us test coverage but not test results